### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by stefania11**
fix the ordered list of steps

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -16,13 +16,15 @@ To learn more about Fixie, check us out at [https://fixie.ai](https://fixie.ai).
...
 
-* 1. Verify you can access [app.fixie.ai](http://app.fixie.ai) with your Google or GitHub email
-* 2. Install the fixie CLI with `pip install fixieai` and run `fixie auth` to ensure your are successfully authenticated
-* 3. Fork the [examples repo](https://github.com/fixie-ai/fixie-examples)
-* 4. Choose any of the example agents, `cd` into the directory, and run `fixie agent deploy`
-* 5. Test that agent by running `fixie console -a username/agent_name` and inputting a test query. You can also run `fixie console` and then @ your specific agent (e.g. `@username/agent_name this is the query`)
-* 6. You can also talk to deployed agents directly at [app.fixie.ai](http://app.fixie.ai)
-* 7. For local development and testing, you can also run your agent locally with `fixie agent serve`. This is easier for debugging what’s happening when things go wrong. This will create a tunnel to your local machine. After running serve, open a new terminal window and talk to the agent just like in step 5.
+<ol>
+  <li>Verify you can access [app.fixie.ai](http://app.fixie.ai) with your Google or GitHub email</li>
```
Comment (petersalas) [inline docs/index.md:20]: It looks like the HTML list breaks the markdown inside?

https://github.com/fixie-ai/fixie-sdk/blob/c067aea8357a0719b61932b2343bcfa328453df9/docs/index.md#getting-started

Comment (petersalas) [reply docs/index.md:20]: I think if you get rid of the `- *` it _should_ work. e.g.

1. test
2. test
3. again

vs.

- * 1. test
- * 2. test
- * 3. again

Comment (stefania11) [reply docs/index.md:20]: That's what we had before. It rendered well locally but not online https://docs.fixie.ai/

Comment (petersalas) [reply docs/index.md:20]: Prior to this change I see
```
* 1. a
* 2. b
```
but I just confirmed via `just serve-docs` that
```
1. a
2. b
```
will work correctly.

Comment (stefania11) [reply docs/index.md]: just updated it

**Review by petersalas — APPROVED** [review]

**Pull request merged**